index.html

```diff
@@ -1,299 +1,318 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">    
-    <link rel="stylesheet" href="./style.css">
-    <title>Samuel Kinyanjui</title>
-</head>
-<body>
-   <section class="myportfolio"> 
-        <section class="toolbar_headline">
-           
-            <header class="headerx">
-                <div class="logox"> <img src="./img/Mylogo.png" alt="Samuel Logo">
-                </div>
-
-                <nav class="navigation">
-                    <img src="./img/menu.png"> 
-                
-                </nav>
-            </header>      
-
-            <section class="headline">            
-                <div class="headerh1">
-                <h1>I’m Samuel k.
-                    Glad to see you!</h1>
-                </div>
-                <div class="headerdetails">
-                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-                    
-                </div>
-                <div class="letconnects">
-                <h4>LET’S CONNECT</h4>  
-                </div>
-                <div class="socialmediaclass">
-                    <img id="socialmedia" src="./img/socialmedia.png">
-                </div>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portfolio</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <nav class="top-menu">
+        <div class="sub-menu">
+          <h1>Samuel Kinyanjui</h1>
+          <ul>
+            <li id="down"><a href="#">Portfolio</a></li>
+            <li id="down"><a href="#">About</a></li>
+            <li id="down"><a href="#">Contact</a></li>
+          </ul>
+          <img src="./image/hamburguer1111.png" alt="humber" id="humb">
+        </div>
+      </nav>
+      <section class="main-bar">
+      <header>
+        <div class="sub-main">
+          <h1>I’m Samuel <br>
+            Glad to see you!</h1>
+          <p>I’m a software developer! I can help you build a product , feature or 
+            website Look through some of my work and experience! If you like what you 
+            see and have a project you need coded, don’t hesitate to contact me.</p>
+            <div class="social">
+              <h4>LET’S CONNECT</h4>
+              <ul>
+                <li>
+                  <a href="#"
+                    ><img src="./image/twitter.png" alt="angellist"
+                  /></a>
+                </li>
+                <li>
```

Comment:
- [ ] Lastly, just as the previous reviewer mentioned your mobile design needs to look as it did previously but currently, it looks all jumbled up. So kindly fix this as well :100: 

- Your Design at 375px currently 

[Screencast from 06-17-2023 11:38:11 AM.webm](https://github.com/samuelkinyanjui/-finish_mobile_version_rev/assets/102699958/8423a117-bab9-4ccb-94a8-1b00cfb0fb40)